clusterrunner_ha.sysconfig

```diff
@@ -0,0 +1,70 @@
+## Type:        string
+## Default:     "cluster-master.dev.box.net"
+## ServiceRestart: clusterrunner_ha
+#
+# The hostname of the proxy for the Cluster Master service.
+#
+CLUSTERMASTER_PROXY_HOSTNAME="master_host.domain.net"
```

Comment:
At Box, we implemented high-availability for ClusterRunner via and nginx proxy. So, for non-HA deployments, you can just set this URL to the same as your cluster-master host's url (MASTER_HOSTS_CSV).